code/modules/antagonists/malf_ai/malf_ai_modules.dm

```diff
@@ -1056,10 +1066,6 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
 		return FALSE
 	var/mob/living/silicon/ai/ai_caller = caller
 
-	if(ai_caller.incapacitated())
-		unset_ranged_ability(caller)
-		return FALSE
```

Comment:
Any reason in particular why this was removed?

Reply:
nah, just a casualty of a merge conflict, ill re-add it